ui/src/app/cohort-search/overview/overview.component.html

```diff
@@ -1,27 +1,30 @@
 <div>
   <div class="overview-header">
-    <h3 id="total-count">
+    <h2 id="total-count">
       Total Count:
       <span *ngIf="(isRequesting$ | async)" class="spinner spinner-sm">
         Loading...
       </span>
       <span *ngIf="!(isRequesting$ | async)">
         {{ total$ | async }}
       </span>
-    </h3>
+    </h2>
     <div>
       <button
         type="button"
-        class="btn btn-success"
+        class="btn btn-primary"
         [style.margin-left]="'0'"
         (click)="saveModal.open()">
-        <clr-icon shape="floppy"></clr-icon>
         Save Cohort
       </button>
     </div>
   </div>
 
-  <app-charts [data]="(chartData$ | async)"></app-charts>
+  <div class="card bg-faded">
```

Comment:
Are you intending to eventually use `/styles/cards.css` for this? (Just to check on that.)

I am worried it may mess with the spacing here, so it may be a good idea to avoid?

Reply:
> I am worried it may mess with the spacing here, so it may be a good idea to avoid?

That's pretty much exactly what happened; it adds more padding (or something) than this element wants, so I just pulled over some of the non-space-related properties.